<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TlasPropertiesPane</class>
 <widget class="QWidget" name="TlasPropertiesPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1392</width>
    <height>1116</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontal_layout_">
   <item>
    <widget class="QScrollArea" name="main_scroll_area_">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="main_content_">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1372</width>
        <height>1096</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vertical_layout_1_">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="ScaledLabel" name="content_node_address_">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="top_vertical_spacer_">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="properties_layout_">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="grid_layout_">
           <property name="horizontalSpacing">
            <number>20</number>
           </property>
           <property name="verticalSpacing">
            <number>1</number>
           </property>
           <item row="0" column="0">
            <widget class="ScaledLabel" name="label_title_nodes_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Node information</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="ScaledLabel" name="label_num_nodes_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Number of nodes:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ScaledLabel" name="content_num_nodes_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="ScaledLabel" name="label_num_box_nodes_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Number of box nodes:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="ScaledLabel" name="content_num_box_nodes_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="ScaledLabel" name="label_num_box16_nodes_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Number of Box16 nodes:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="ScaledLabel" name="content_num_box16_nodes_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="ScaledLabel" name="label_num_box32_nodes_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Number of Box32 nodes:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="ScaledLabel" name="content_num_box32_nodes_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="ScaledLabel" name="label_num_instances_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Number of instance nodes:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="ScaledLabel" name="content_num_instances_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="ScaledLabel" name="label_num_blases_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Number of unique BLASes:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="ScaledLabel" name="content_num_blases_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="ScaledLabel" name="label_num_triangles_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Number of triangles:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="ScaledLabel" name="content_num_triangles_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="ScaledLabel" name="label_num_procedural_nodes_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Number of procedural nodes:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="ScaledLabel" name="content_num_procedural_nodes_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <spacer name="vertical_spacer_table_1_">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="10" column="0">
            <widget class="ScaledLabel" name="label_title_build_flags_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Build flags</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="ScaledLabel" name="label_allow_update_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Allow update:</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="ReadOnlyCheckBox" name="content_allow_update_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="ScaledLabel" name="label_allow_compaction_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Allow compaction:</string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="ReadOnlyCheckBox" name="content_allow_compaction_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="ScaledLabel" name="label_low_memory_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Low memory:</string>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="ReadOnlyCheckBox" name="content_low_memory_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <widget class="ScaledLabel" name="label_build_type_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Build type:</string>
             </property>
            </widget>
           </item>
           <item row="14" column="1">
            <widget class="ScaledLabel" name="content_build_type_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="15" column="0">
            <spacer name="vertical_spacer_table_2_">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="16" column="0">
            <widget class="ScaledLabel" name="label_title_memory_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Memory usage</string>
             </property>
            </widget>
           </item>
           <item row="17" column="0">
            <widget class="ScaledLabel" name="label_memory_tlas_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>TLAS:</string>
             </property>
            </widget>
           </item>
           <item row="17" column="1">
            <widget class="ScaledLabel" name="content_memory_tlas_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="18" column="0">
            <widget class="ScaledLabel" name="label_memory_total_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Total:</string>
             </property>
            </widget>
           </item>
           <item row="18" column="1">
            <widget class="ScaledLabel" name="content_memory_total_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontal_spacer_">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="bottom_vertical_spacer_">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScaledLabel</class>
   <extends>QLabel</extends>
   <header>qt_common/custom_widgets/scaled_label.h</header>
  </customwidget>
  <customwidget>
   <class>ReadOnlyCheckBox</class>
   <extends>QCheckbox</extends>
   <header>views/custom_widgets/read_only_checkbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
